TestFoundation/TestCodable.swift

```diff
@@ -404,8 +404,8 @@ class TestCodable : XCTestCase {
         for calendar in calendarValues {
             do {
                 try expectRoundTripEqualityThroughJSON(for: calendar)
-            } catch let error {
                 XCTFail("\(error) for \(calendar)")
+            } catch {
```

Comment:
Should the `catch` not be 2 lines up?

Reply:
Yes, that is correct. I messed up with another changed I made while I was picking the changes to `stage` before commit. Nice catch :)

Reply:
Should be fine now.